### PR TITLE
DSS-1271 Add ping database function

### PR DESCRIPTION
**Pull request by maxmilazzo**
https://timeoutgroup.atlassian.net/browse/DSS-1271

Added function to test database status

I will add a commit to remove Bintray usage and use Nexus instead

### CODE REVIEW

File: src/test/scala/com/timeout/healthchecks/HeathChecksTest.scala [hunk 1]
```diff
@@ -49,6 +49,13 @@ class HeathChecksTest extends AnyFunSpec with ScalaFutures with Matchers with Ta
   }
 
   describe("Ping check") {
+
+    it("should return ok for valid database ping") {
+      whenReady(HealthChecks.pingDataBase("sdb.gp-db.dev.timeout.com", 3306, "sdbuser", "thepword").run()) { r =>
```
Comment (kthhrv) [inline src/test/scala/com/timeout/healthchecks/HeathChecksTest.scala]: this test may not have access to sdb.gp-db.dev.timeout.com, test should be mocked?

Comment (maxmilazzo) [reply src/test/scala/com/timeout/healthchecks/HeathChecksTest.scala]: I may just commented out. We can just run locally if needed. The code is simple so I was just a way to confirm the chnages

File: devops/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Read by https://github.com/timeoutdigital/platform/tree/master/lambda/codebuild_notifications
+to determine where to send notifications for failed Master builds.
+
+
+publish-buildspec.yml 
+---------------------
+
+Run by AWS CodePipeline when a GitHub release is created, status can be viewed at:
+https://eu-west-1.console.aws.amazon.com/codesuite/codebuild/511042647617/projects/scala-commons-publish
```
Comment (maxmilazzo) [inline devops/README.md:12]: this needs update

File: src/test/scala/com/timeout/healthchecks/HeathChecksTest.scala (hunk 1 above)
Comment (kthhrv) [reply src/test/scala/com/timeout/healthchecks/HeathChecksTest.scala]: I've removed the test

**Comment by llewellynthomas** [comment]
@maxmilazzo Hello, I hope you are all well :) Just letting you know this is public.

But this is not:
```
"MySql driver" at "https://nexus.timeout.com/content/repositories",
```

**Comment by maxmilazzo** [comment]
@llewellynthomas this changes are no longer, I am closing the PR.

I hope you are well to, all the best

**Pull request closed without merging**